test/components/views/right_panel/RoomHeaderButtons-test.tsx

```diff
@@ -52,7 +56,7 @@ describe("RoomHeaderButtons-test.tsx", function () {
         return container.querySelector(".mx_RightPanel_threadsButton");
     }
 
-    function isIndicatorOfType(container, type: "red" | "gray") {
+    function isIndicatorOfType(container: React.ReactElement, type: "red" | "gray" | "bold") {
```

Comment:
```suggestion
    function isIndicatorOfType(container: HTMLElement, type: "red" | "gray" | "bold") {
```

Reply:
Ah, thanks! I couldn't figure out the proper type here, but the strict any check was complaining. 👍